fhir-search/src/test/java/com/ibm/fhir/search/test/ChainParameterParseTest.java

```diff
@@ -308,6 +309,9 @@ public void testValidChainWithDateSearchParameter() throws Exception {
         FHIRSearchContext searchContext;
         Class<ClinicalImpression> resourceType = ClinicalImpression.class;
         Instant now = Instant.now().truncatedTo(ChronoUnit.MILLIS);
+        if (now.get(ChronoField.MILLI_OF_SECOND) == 0) {
```

Comment:
I think this deserves a comment as to why it's necessary. Perhaps it's just the spec, but having to adjust times like this makes me worry about our date-time handling.

Reply:
Comment added.